completions/bind9

```diff
@@ -0,0 +1,198 @@
+# bash completion for nslookup                             -*- shell-script -*-
+# bind9 utilities completion
+
+_comp_cmd_rndc__list_commands()
+{
+	rndc 2>&1 | awk '/^  / {print $1}' | sort -u
+}
+
+_comp_cmd_named_checkconf__list_zones()
+{
+	named-checkconf -l | awk '{print $1}'
+}
+
+_comp_cmd_rndc__list_parameters()
+{
+	local SUBCMD=$1
+	rndc 2>&1 | awk "/^  ${SUBCMD} / { print \$2 }"
+}
+
+_comp_cmd_rndc()
+{
+	local cur prev words cword comp_args
+	_comp_initialize -n = -- "$@" || return
+
+	case $prev in
+		-c|-k)
+			_comp_compgen filedir
```

Comment:
Thanks for updating!

```suggestion
			_comp_compgen_filedir
```

The above two versions are functionally equivalent, but the latter is required by our coding style:

https://github.com/scop/bash-completion/blob/af8a80d8916edd4a1c146afb2743651b6c3ca406/doc/api-and-naming.md?plain=1#L148-L149

This applies to the other calls of `_comp_compgen filedir` as well.